service/rolesService.ts

```diff
@@ -0,0 +1,15 @@
+const axios = require('axios');
+import { Roles } from "../model/Roles";
+axios.defaults.baseURL = process.env.API_URL;
```

Comment:
`API_URL` will need added to the GitHub Action for deployments. This should also be documented in the `README.md` file